src/go/configmanager/flags/flags.go

```diff
@@ -69,6 +69,10 @@ var (
   omitted, the proxy contacts the metadata service to fetch an access token`)
 	TokenAgentPort = flag.Uint("token_agent_port", 8791, "Port that configmanager use to setup server to provide envoy with access token using service account credential, for accessing servicecontrol.")
 
+	// Flags for external calls.
+	DisableOidcDiscovery = flag.Bool("disable_oidc_discovery", false, `Disable OpenID Connect Discovery, prevents config generator from making external calls to determine the JWKS URI. 
```

Comment:
If enabled, the config generator will make external calls ...

Reply:
Sorry, could you clarify what you mean. I clarified what disabling oidc would do.